src/components/Layout/index.tsx

```diff
@@ -31,7 +31,7 @@ const IndexLayout = ({
             >
               📣 GraphQL Conf 2023 • Sept 19-21 • San Francisco •{" "}
               <span className="underline underline-offset-2">
-                Register Now!
+                Watch the videos!
```

Comment:
Might be worth linking directly to the sessions: https://graphql.org/conf/sessions/ or youtube playlist https://www.youtube.com/watch?v=3zLmU-W2wYA&list=PLP1igyLx8foE9SlDLI1Vtlshcon5r1jMJ